hbase-protocol-shaded/src/main/protobuf/Admin.proto

```diff
@@ -369,6 +369,9 @@ service AdminService {
   rpc GetSlowLogResponses(SlowLogResponseRequest)
     returns(SlowLogResponses);
 
+  rpc GetLargeLogResponses(SlowLogResponseRequest)
```

Comment:
same question as above.. we can get rid of this one, right? (I thought that was our discussion in the PR earlier).

Reply:
We discussed about getting rid of separate Admin API and have a same endpoint. This one is for RS RPC services. Since we don't want to have multiple enums in Admin.proto, TooSlowLogs.proto, we need this separately and it's only used in RSRpcServices with minimal code.

Reply:
Another reason why we must have this is to find out sometimes which specific Get API was slow or large in nature even between get_slow and get_large responses.

Sample response:
```
{
  "startTime": 1585735414657,
  "processingTime": 1,
  "queueTime": 0,
  "responseSize": 2351,
  "clientAddress": "171.30.20.8:63571",
  "serverClass": "HRegionServer",
  "methodName": "GetLargeLogResponses",
  "callDetails": "GetLargeLogResponses(org.apache.hadoop.hbase.shaded.protobuf.generated.AdminProtos$SlowLogResponseRequest)",
  "param": "class org.apache.hadoop.hbase.shaded.protobuf.generated.AdminProtos$SlowLogResponseRequest",
  "userName": "vjasani"
}
{
  "startTime": 1585735372313,
  "processingTime": 1,
  "queueTime": 0,
  "responseSize": 2260,
  "clientAddress": "171.30.20.8:63571",
  "serverClass": "HRegionServer",
  "methodName": "GetSlowLogResponses",
  "callDetails": "GetSlowLogResponses(org.apache.hadoop.hbase.shaded.protobuf.generated.AdminProtos$SlowLogResponseRequest)",
  "param": "class org.apache.hadoop.hbase.shaded.protobuf.generated.AdminProtos$SlowLogResponseRequest",
  "userName": "vjasani"
}
```